src/frontends/ir/README.md

```diff
@@ -0,0 +1,45 @@
+# OpenVINO IR Frontend
+
+```mermaid
+flowchart LR
+    ir[("IR (*.xml)")]
+        
+    style ir fill:#427cb0
+
+    ir_fe["OpenVINO IR Frontend"]
+    
+    openvino(openvino library)
+    ir--Read ir---ir_fe
+    ir_fe--Create ov::Model--->openvino
+    click ir "https://docs.openvino.ai/latest/openvino_docs_MO_DG_IR_and_opsets.html"
+```
+
+The main responsibility of OpenVINO IR Frontend is creation `ov::Model` representation from OpenVINO IR.
```

Comment:
people sometimes are asking whether ov::Model is OpenVINO IR or not? I suppose ov::Model is in-memory IR, so the statement can be corrected then